Makefile.am

```diff
@@ -330,6 +330,8 @@ EXTRA_DIST = $(TUTORIAL_DIST) \
              scripts/outlier/README.md \
              scripts/outlier/outlier_detection.py \
              scripts/outlier/theta_nodelist_broken.txt \
+             specs/geopm-gcc-mvapich2-quartz.spec \
```

Comment:
Since the intended use of these spec files is to build GEOPM RPMs for TOSS (not Quartz specifically), you could change the spec file names from *-quartz to *-toss (or no suffix). I was successfully able to build RPMs on Catalyst using the same spec files. We assume that we should be able to build RPMs on other systems running TOSS since all of them have the same root directory structures.